sql/0000.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS tbl_phonebook(
+    first_name VARCHAR(100) NOT NULL,
+	last_name VARCHAR(100) NOT NULL,
+    address VARCHAR(100) NOT NULL,
+    phone_number VARCHAR(50) PRIMARY KEY NOT NULL,
+    email VARCHAR(100)
+)
+
+CREATE TABLE IF NOT EXISTS tbl_schedule(
+    class_id SERIAL PRIMARY KEY,
+    date DATE NOT NULL,
+    lesson_one VARCHAR(50),
+    lesson_two VARCHAR(50),
+    lesson_three VARCHAR(50),
+    lesson_four VARCHAR(50)
+)
+
+CREATE TABLE IF NOT EXISTS tbl_login(
+    user_name VARCHAR(50) NOT NULL,
+    date DATE NOT NULL,
+    login TIMESTAMP NOT NULL,
+    logout TIMESTAMP NOT NULL
```

Comment:
what the value of `logout` when user signed-in?

Reply:
Maybe change to
activity_time TIMESTAMP NOT NULL,
action_type VARCHAR(50) NOT NULL
Than act_type values will be 'login' or 'logout'

Reply:
> what the value of `logout` when user signed-in?

 Maybe change to activity_time TIMESTAMP NOT NULL, action_type VARCHAR(50) NOT NULL Than act_type values will be 'login' or 'logout'